### PR TITLE
Update schema & extensions page

### PR DESCRIPTION
**Pull request by EliLawrence**
* moved the what's an ID section higher on the page
* edited text describing the different extensions to try to make it more concise
* clarified guidelines for measurement ID fields and updated some relevant links
* shortened the table for eMoF terms so only most relevant shown
* removed the tables for Event & Occurrence terms

### CODE REVIEW

File: _episodes/04-create-schema.md [hunk 1]
```diff
@@ -33,100 +33,64 @@ The Occurrence core allows you to provide all the required Darwin Core terms det
...
 
 > ## Tip 
-> You can search for `measurementType` that other OBIS data providers have used by using the [OBIS mof report](https://mof.obis.org/). BE CAREFUL though to make sure the
-> definition in the URI matches exactly your measurement type before you reuse it for your data.
+> You can search for `measurementTypes` that other OBIS data providers have used by using the [OBIS mof report](https://mof.obis.org/). BE CAREFUL though to make sure the
```
Comment (7yl4r) [inline _episodes/04-create-schema.md:50]: I am probably in the pedantic minority here but I would format this like `measurementType`s

**Review by 7yl4r — APPROVED** [review]

**Comment by MathewBiddle** [comment]
The changes look good. I want to build this locally before we merge to make sure the MD -> Jekyll works as expected.

**Comment by MathewBiddle** [comment]
Everything build and functions as expected. Thanks @EliLawrence! 

I'm curious about the removal of the event and occurrence core table dropdowns. I'm curious about your thought process on removing those tables. Is it because some of that material is presented in the introduction?

Here is the section on the current page:
image

Here is the changed page:
image

**Review by jdpye — APPROVED** [review]
Changes look good to me, I'm glad Mat's building these locally though, that's the piece i'm missing at the moment for evaluating whether these look good in the site. I'm using the rich diffs to approximate the markdown as a temporary measure.

**Comment by MathewBiddle** [comment]
> Can we reject the removal of the tables but keep the others?

Let me try something...

**Comment by MathewBiddle** [comment]
I think I did it:

image

I used pycharm to compare your PR with the gh-pages branch then used the accept feature to merge back in the original tables 
image
very convoluted and probably more clear to just copy and paste from the source. But, I learned something today 🤷

**Review by MathewBiddle — APPROVED** [review]
This looks good. Thanks @EliLawrence

**Comment by MathewBiddle** [comment]
Merging. Take a look in ~5 minutes to see the changes on the website.

**Pull request merged**